packages/api/src/controllers/playback.ts

```diff
@@ -184,7 +213,10 @@ app.get("/:id", async (req, res) => {
   const ingest = ingests[0].base;
 
   let { id } = req.params;
-  const info = await getPlaybackInfo(req, ingest, id);
+  const isEmbeddablePlayer = embeddablePlayerOrigin.test(
+    req.headers["origin"] ?? ""
```

Comment:
This could be spoofed on applications outside of the browser to do a cross user query on CIDs, but we probably don't mind

Reply:
Yeah it's more about billing the right user than protecting any kind of sensitive info.